src/sentry/snuba/discover.py

```diff
@@ -608,7 +610,8 @@ def get_facets(query, params, limit=10, referrer=None):
             limit=TOP_VALUES_DEFAULT_LIMIT,
             dataset=Dataset.Discover,
             referrer=referrer,
-            sample=sample_rate,
```

Comment:
@fpacifici why is `sample=sample_rate` removed here?

Reply:
copy paste mistake